utils/exceptions.py

```diff
@@ -15,28 +15,31 @@
 """
 
 class VFATDACBiasCannotBeReached(ValueError):
-    """
-    This exception is raised if DAC analysis determines that to achieve the 
-    nominal bias current/voltages in ``nominalDacValues`` dictionary, 
-    of: ``gempython.gemplotting.utils.anaInfo``, the DAC would need to be set
-    to a value outside of the DAC range ``[0, max]`` where ``max`` is
-    ``maxVfat3DACSize[dacSelect][0]``.  Info can be found on these ranges in
-    the ``maxVfat3DACSize`` dictionary of: ``gempython.tools.hw_constants``
-    """
```

Comment:
If the the class is still destined to remain, why is its docstring removed?

Reply:
the class is removed